website/README.md

```diff
@@ -47,6 +47,7 @@ Use Jekyll to serve the website locally (by default, at
 `http://localhost:8080`):
 
 ```sh
+$ make js
```

Comment:
Can you remove this? It's unrelated and also implied by the `make` two lines below.